content/java/concepts/math-methods/terms/tanh/tanh.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: '.tanh()'
+Description: 'Returns the hyperbolic sine of a double-type value.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.tanh()`** method returns the hyperbolic sine of a `double`-type value.
+
+## Syntax
+
+```pseudo
+Math.tanh(double num)
+```
+
+The tanh() method takes a single parameter.
+
+`num` - value of an angle whose hyperbolic tangent is to be determined
```

Comment:
```suggestion
`num` is the value of an angle whose hyperbolic tangent is to be determined.
```

Reply:
Since it's just one point here, we'll convert into a sentence. If there were more points, we would turn this into a bulleted list and then use a colon to separate a term and its definition or explanation